django_mock_queries/query.py

```diff
@@ -174,9 +175,11 @@ def __iter__():
     mock_set.__iter__ = MagicMock(side_effect=__iter__)
 
     def create(**attrs):
+        validate_mock_set(mock_set)
+        for k in attrs.keys():
+            if k not in [f.attname for f in mock_set.model._meta.concrete_fields]:
+                raise ValueError('MockSet model has no field {}'.format(k))
```

Comment:
Can we raise in this case a `TypeError: 'foo' is an invalid keyword argument for this function`, like Django, see `django/db/models/base.py` - line 555?